recipe/meta.yaml

```diff
@@ -54,7 +54,9 @@ test:
     - test -f $PREFIX/lib/libfreeimage.{{ version.split('.')[0] }}${SHLIB_EXT}  # [osx]
     - test -f $PREFIX/lib/libfreeimage-{{ version }}${SHLIB_EXT}  # [unix]
     - test -f $PREFIX/include/FreeImage.h  # [unix]
-
+    - if exist %PREFIX%\\Library\\include\\FreeImage.h (exit 0) else (exit 1)  # [win]
```

Comment:
I think the inverse is actually better `if not exist ... exit 1` because IIRC this will exit the script before executing the next two lines.

Reply:
Good point, just for reference I was [cargo-culting](https://en.wikipedia.org/wiki/Cargo_cult_programming) from https://github.com/conda-forge/libignition-math4-feedstock/blob/master/recipe/meta.yaml#L38.

Reply:
So I am afraid a lot of feedstock are affected by the same problem.

Reply:
yep ... it's part of the reason I wanted to make this simpler with boa.